libkpod/containerserver.go

```diff
@@ -113,7 +120,147 @@ func New(config *Config) (*ContainerServer, error) {
 			sandboxes:  make(map[string]*sandbox.Sandbox),
 		},
 		config: config,
-	}, nil
+	}
+
+	return c, c.Update()
+}
+
+// Update makes changes to the server's state (lists of pods and containers) to
+// reflect the list of pods and containers that are stored on disk, possibly
+// having been modified by other parties
+func (c *ContainerServer) Update() error {
```

Comment:
We need that for create anyways, I already have a commit for that. Let me PR and and you can rebase off it.